src/entrypoints/main/router/pages/datasets-page/index.tsx

```diff
@@ -168,6 +178,15 @@ const DatasetsPage: FC<Props> = ({
               placeholder='Søk etter datasett'
               onSubmit={searchSubmit}
               onClear={searchClear}
+              onChange={e => {
+                setSearchString(e.target.value);
+                if (!cachedSuggestions?.[e.target.value]) {
```

Comment:
Jeg tror uansett ikke du får noe treff på bare ett enkelt tegn, så du kan ta med en betingelse at lengden på søkeordet må være > 1 for at det gjøres et kall.